### ISSUE TITLE
Doubt on using tfa.optimizers.MovingAverage

### ISSUE THREAD

**Issue by heytitle** [tensorflow/addons#741]
**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Colab
- TensorFlow version and how it was installed (source or binary): 2.0.0
- TensorFlow-Addons version and how it was installed (source or binary): 0.6.0
- Python version: 3.6.0
- Is GPU used? (yes/no): N/A

**Describe the bug**
Would you mind clarify how one should use MovingAverage? Possibly, this is not a bug and it's just that I don't know to use it. 

This is the way I use it: `tfa.optimizers.MovingAverage(tf.keras.optimizers.SGD(0.001))`. When running training, it seems no gradient update occurred: loss is at the same value.

**Code to reproduce the issue**
You can find my code here: https://colab.research.google.com/drive/1Q_Jgkf69dUXhfXli4IXANGjUYdZz2xeU#scrollTo=GEZmiZv56Cen.

**Comment by WindQAQ** [comment]
cc @Squadrick

**Comment by Squadrick** [comment]
Thanks for raising this issue. It isn't a fault of usage, but a bug in the code due to changes to `tf.train.ExponentialMovingAverage`. I've submitted a PR that should fix this.

**Comment by heytitle** [comment]
Thanks for the fix!

**Comment by Squadrick** [comment]
@heytitle What do you think about the current API for `MovingAverage`? Is the `assign_average_vars()` mechanism fine or would you prefer something else?

**Comment by heytitle** [comment]
I think it's fine.
